CONTRIBUTING.md

```diff
@@ -118,6 +118,75 @@ private:
+| [Chinese](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=zh_CN) | CloudiDust | cloudidustATgmailDOTcom |
+| [Chinese (Taiwan)](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=zh_TW) | 楊綮銘 | steven2880ATgmailDOTcom |
+| [Croatian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=hr) | None | NA |
+| [Czech](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=cs) | Michal Žejdl | lachimATemerDOTcz |
+| [Danish](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=da) | None | NA |
+| [Dutch](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=nl) | Merijn de Vet | tybonzerATliveDOTnl |
+| [English (Shavian chars)](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=en@shaw) | NA | NA |
+| [Esperanto](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=eo) | Luther Thompson | lutherATlibremDOTone |
+| [Estonian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=et) | None | NA |
+| [Filipino](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=tl) | None | NA |
+| [Finnish](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=fi) | Jaakko Saarikko (styxnix) | jaakkoDOTsaarikkoATprotonmailDOTcom |
+| [French](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=fr) | Mathieu Guilbaud (Guim) | NA |
+| [Galician](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=gl) | Adrián Chaves Fernández (Gallaecio) | adriyetichavesATgmailDOTcom |
+| [German](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=de) | Aaron Winter (Bitron) |  |
+| [Greek](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=el) | None | NA |
+| [Hebrew](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=he) | None | NA |
+| [Hungarian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=hu) | Széll András | szellDOTandrisATgmailDOTcom |
+| [Icelandic](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=is) | None | NA |
+| [Indonesian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=id) | Irsyad Musthafa | sevennightmareATtutanotaDOTde |
+| [Irish](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=ga) | None | NA |
+| [Italian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=it) | Antonio Rosella | arosellaATyahooDOTcom |
+| [Japanese](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=ja) | Hironori Fujimoto (RatArmy) | broadbarredfirefishATgmailDOTcom |
+| [Korean](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=ko) | mistzone | drier22ATgmailDOTcom |
+| [Latin](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=la) | None | NA |
+| [Latvian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=lv) | None | NA |
+| [Lithuanian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=lt) | Andrius Štikonas | stikonasATgmailDOTcom |
```

Comment:
Last active in January 2019. https://forums.wesnoth.org/memberlist.php?mode=viewprofile&u=109204